ctapipe/core/qualityquery.py

```diff
@@ -93,7 +98,7 @@ def __str__(self):
         """Print a formatted string representation of the entire table."""
         return str(self.to_table())
 
-    def __call__(self, **kwargs) -> np.ndarray:
+    def __call__(self, tel_id, **kwargs) -> np.ndarray:
```

Comment:
Since the quality query supports being called with the telescope type, I think we should also support this here. E.g. do not use `tel_id` but `key: [int, str, TelescopeDescription]`?

Reply:
And if you use `key=None`, the API is backwards compatible,i.e. `__call__(...)` without a key will take the global default value, which is what the current configurations will be parsed into.